.chglog-default/CHANGELOG.tpl.md

```diff
@@ -0,0 +1,36 @@
+{{ range .Versions }}
+<a name="{{ .Tag.Name }}"></a>
+## {{ if .Tag.Previous }}[{{ .Tag.Name }}]{{ else }}{{ .Tag.Name }}{{ end }} - {{ datetime "2006-01-02" .Tag.Date }}
+{{ range .CommitGroups -}}
+### {{ .Title }}
+{{ range .Commits -}}
+- {{ if .Scope }}**{{ .Scope }}:** {{ end }}{{ .Subject }} ([#{{ .Hash.Short }}]({{$.Info.RepositoryURL}}/commits/{{.Hash.Short}}))
+{{ range .Notes -}}
+{{ if contains .Title "BREAKING CHANGE" -}}
+{{indent "```" 4}}
+{{indent .Title 4}}:
+{{indent .Body 4}}
+{{indent "```" 4}}
+{{ end -}}
+{{ end -}}
+{{ end -}}
+{{ end -}}
+
+{{- if .RevertCommits -}}
+
+### Reverts
+{{ range .RevertCommits -}}
+- {{ .Revert.Header }}
+{{ end }}
+{{ end -}}
+
+{{ end -}}
+
+{{- if .Versions }}
+[Unreleased]: {{ .Info.RepositoryURL }}/compare/{{ $latest := index .Versions 0 }}{{ $latest.Tag.Name }}...HEAD
+{{ range .Versions -}}
+{{ if .Tag.Previous -}}
+[{{ .Tag.Name }}]: {{ $.Info.RepositoryURL }}/compare/{{ .Tag.Previous.Name }}...{{ .Tag.Name }}
+{{ end -}}
+{{ end -}}
+{{ end -}}
```

Comment:
holy smokes, will need to come up to speed on what all this means. :)

Reply:
This is Go templates https://blog.gopheracademy.com/advent-2017/using-go-templates/.

Reply:
First time here too, I had the same reaction :)   Once I dug into it, it was kind of fun.